Assets/MRTK/Examples/Demos/EyeTracking/General/Scripts/TargetBehaviors/OnLoadStartScene.cs

```diff
@@ -6,19 +6,44 @@
 
 namespace Microsoft.MixedReality.Toolkit.Examples.Demos.EyeTracking
 {
+
     /// <summary>
     /// When the button is selected, it triggers starting the specified scene.
     /// </summary>
     [AddComponentMenu("Scripts/MRTK/Examples/OnLoadStartScene")]
     public class OnLoadStartScene : MonoBehaviour
     {
+        private enum LoadOptions
+        {
+            LoadOnDeviceAndInEditor,
+            LoadOnlyOnDevice,
+        }
         [SerializeField]
         [Tooltip("Name of the scene to be loaded when the button is selected.")]
         private string SceneToBeLoaded = "";
 
+        [SerializeField]
+        [Tooltip("Option to only load the scene if running on the HoloLens device.")]
+        private LoadOptions LoadOption = LoadOptions.LoadOnlyOnDevice;
```

Comment:
Only other comment would be around [casing of private fields](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#naming-conventions). We typically use camelCase.

Reply:
Ah good point. @sostel  FYI regarding this coding convention in MRTK. Leaving as is to stay consistent with existing coding style in the file. I think most of the eye tracking samples might capitalize the first letter. It's a good question whether we should ask sample contributors to follow our coding standards. I would vote to make all the code consistent, and asking future contributors to follow one consistent style.